rainier-scalacheck/src/main/scala/com/stripe/rainier/scalacheck/package.scala

```diff
@@ -1,37 +1,27 @@
 package com.stripe.rainier.scalacheck
 
-import com.stripe.rainier.core.Generator
-import com.stripe.rainier.core.RandomVariable
+import com.stripe.rainier.core.{Categorical, Generator, RandomVariable}
 import com.stripe.rainier.compute.Real
-import com.stripe.rainier.sampler.RNG
-import com.stripe.rainier.sampler.ScalaRNG
-
-import org.scalacheck.Arbitrary
-import org.scalacheck.Arbitrary.arbitrary
-import org.scalacheck.Cogen
-import org.scalacheck.Gen
+import com.stripe.rainier.sampler.{RNG, ScalaRNG}
+import org.scalacheck.{Arbitrary, Cogen, Gen}
 
 object `package` {
+  import Arbitrary.arbitrary
 
-  implicit val arbitraryReal: Arbitrary[Real] =
-    Arbitrary(arbitrary[Double].map(Real(_)))
-
-  implicit def arbitraryGenerator[A: Arbitrary]: Arbitrary[Generator[A]] =
-    Arbitrary(genGenerator)
+  implicit val cogenRNG: Cogen[RNG] =
```

Comment:
I moved this out of `genGenerator`, thinking I would need it... I don't, but it's useful outside, yeah?